src/settings/dto/create-setting.dto.ts

```diff
@@ -0,0 +1,6 @@
+export class CreateSettingDto {
+  readonly name: string;
+  readonly data_type: string;
```

Comment:
shouldn't data-type be an enum?

Reply:
just realised this is how it's written in docs. I have updated the docs to point that we need an enum